docs/faq.rst

```diff
@@ -65,7 +65,13 @@ I get import errors on libraries that depend on C modules
 
 This happens because our build system doesn't have the dependencies for building your project. This happens with things like libevent and mysql, and other python things that depend on C libraries. We can't support installing random C binaries on our system, so there is another way to fix these imports.
 
-You can mock out the imports for these modules in your ``conf.py`` with the following snippet::
+If you are mocking your dependencies without using the mock library, then you can use this feature `autodoc_mock_imports`_ to mock your dependencies.
```

Comment:
I think what we're wanting to say here is "The recommended method of mocking your libraries is to use the autodoc_mock_imports config option", not "if you are .."

Reply:
Yes I am changing it!

Reply:
@agjohnson I think its fine as it is the syntax for the removing the naked link.

Reply:
What Anthony says is that we should encourage people to use the autodoc_mock_imports method